src/TestUtils/GcloudWrapper/CloudFunction.php

```diff
@@ -181,20 +181,29 @@ public function deploy(array $flags = [], string $trigger = self::DEFAULT_TRIGGE
     /**
      * Delete the deployed function.
      *
+     * @param array $flags optional flags to inject into the deploy command
      * @param int $retries optional The number of retries upon failure
      *
      * @return bool true if the function is succesfully deleted, otherwise false
      */
-    public function delete($retries = 3)
+    public function delete(array $flags = [], $retries = 3)
```

Comment:
we need to make sure we're not setting this anywhere. My guess is that we aren't, and so changing the order is fine.